src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -311,6 +313,48 @@ public async Task HandleGetObjectsFromTSqlModelRequest(GetObjectsFromTSqlModelPa
             return;
         }
 
+        /// <summary>
+        /// Handles request to save a publish profile
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleSavePublishProfileRequest(SavePublishProfileParams parameters, RequestContext<ResultStatus> requestContext)
+        {
+            try
+            {
+                if (parameters.ProfilePath != null)
+                {
+                    DacProfile profile = new DacProfile();
+                    profile.TargetDatabaseName = parameters.DatabaseName;
+                    profile.TargetConnectionString = parameters.ConnectionString;
+                    //TODO: Set deploy options to pass on to DacFx
```

Comment:
Will this happen in this PR or a future one?

Reply:
This is for the future PR.